Boolean_set_operations_2/doc/Boolean_set_operations_2/CGAL/Boolean_set_operations_2.h

```diff
@@ -1,833 +1,2166 @@
 namespace CGAL {
 
-/*!
-\addtogroup boolean_complement Complement Functions
-\ingroup PkgBooleanSetOperations2Ref
-\anchor ref_bso_complement
+/*! \addtogroup boolean_complement Complement Functions
+ * \ingroup PkgBooleanSetOperations2Ref
+ * \anchor ref_bso_complement
+ *
+ * There are several overloaded function templates called `complement()` that
```

Comment:
```suggestion
 * There are several overloaded template functions called `complement()` that
```

Reply:
Nop, it should be "function template"; see, e.g., https://en.cppreference.com/w/cpp/language/function_template